mitodl/mit-open
# Fix linting on nextjs branch

### What are the relevant tickets?

### Description (What does it do?)
On the current NextJS branch, linting is failing with
```
Oops! Something went wrong! :(

ESLint: 8.38.0

TypeError: Failed to load plugin '@typescript-eslint' declared in '.eslintrc.js » eslint-config-mitodl#overrides[0]': Class extends value undefined is not a constructor or null
```

This PR resolves that issue and fixes linting/formatting errors that are present in the initial nextjs work.

### How can this be tested?
1. Run `yarn lint-check` everything should pass.
    - there will be some warnings about `img`
2. The linting check should be passing on CI (typechecking + tests still fail)

### Additional Info
